lib.typ

```diff
@@ -263,6 +263,10 @@
   ]
 
   show heading.where(level: 2): it => {
+    set block(
```

Comment:
`set block` doesn't seem to do anything? I've messed around with the `above` and `below` values and they've produced identical results. Setting `fill: blue`, which should colour the entire block, also does nothing. I'm not fluent in typst yet but it seems to me it should be used like on line 54.

Reply:
This also applies to the other uses of `set block` in this commit, as far as I can tell.

Reply:
Hmm very interesting, it seems like you're right. If I remove all the `block` calls, things still look the same...